browser-extension/mv3/src/service-worker/services/utils.ts

```diff
@@ -187,3 +196,22 @@ export const getAppTabs = async (): Promise<chrome.tabs.Tab[]> => {
 
   return appTabs;
 };
+
+export const sendMessageToApp = async (messageObject: unknown) => {
+  const appTabs = await getAppTabs();
+  return Promise.all(appTabs.map(({ id }) => chrome.tabs.sendMessage(id, messageObject)));
+};
```

Comment:
Do we need to move this here after moving `updateActivationStatus` ? Isn't `updateActivationStatus` enough to break the chain?

Reply:
Yes. Its necessary otherwise `utils` <-> `messageHandler` dependency will be present because `updateActivationStatus` depends on `sendMessageToApp`